File: Java/src/FindLargest.java
import java.util.Arrays;

public class FindLargest {

    public static void main(String[] args) {
        int [] numbers = {25,3, 5, 7, 2, 8, -1, 4,85};
        System.out.println(findLargest(numbers)); // Example usage
    }

    static int findLargest(int[] numbers) {
        if (numbers == null || numbers.length == 0) {
            return 0;
        }

        return Arrays.stream(numbers).max().getAsInt();
    }
}

File: Java/kotlin/FindLargest.kt
package kotlin

object FindLargest {
    @JvmStatic
    fun main(args: Array<String>) {
        val numbers = intArrayOf(25, 3, 5, 7, 2, 8, -1, 4, 85)
        println(findLargest(numbers)) // Example usage
    }

    fun findLargest(numbers: IntArray?): Int {
        if (numbers == null || numbers.isEmpty()) {
            return 0
        }

        return numbers.maxOrNull() ?: 0
    }
}

